sasjs-tests/src/testSuites/Basic.ts

```diff
@@ -61,8 +61,8 @@ export const basicTests = (
         'Should fail on first attempt and should log the user in on second attempt',
       test: async () => {
         await adapter.logOut()
-        await adapter.logIn('invalid', 'invalid', 'clientID1')
-        return await adapter.logIn(userName, password, 'clientID1')
+        await adapter.logIn('invalid', 'invalid').catch((err: any) => {})
```

Comment:
why do we ignore error in this case?

Reply:
We don't need it, we only need to catch it so execution doesn't stop.

This test purpose:
- Login with bad credentials
- Then login with correct and make sure it worked.

Reason: in the past we had issues where when user try to log in with bad credentials, after that, login didn't work until page realod.